docs/UserMenu.md

```diff
@@ -0,0 +1,107 @@
+# User Menu
+
+The `<pxb-user-menu>` is an Avatar that opens a Menu when clicked. It is typically used in the top-right corner of an application and indicates who is logged in.
+
+<div style="align-items: center; display:flex; justify-content: space-around">
+
+<img width="30%" alt="UserMenu Avatar" src="./images/userMenuAvatar.png">
+<img width="35%" alt="UserMenu Opened" src="./images/userMenuOpened.png">
+
+</div>
+
+The Menu can be populated via the `menuGroups` prop, or can be entirely customized by supplying your own content.  See the API section below for more details.
+
+## Usage
+
+<div style="width: 100%; text-align: center">
+    <img width="100%" style="max-width: 500px" alt="User Menu Anatomy" src="./images/userMenuAnatomy.png">
+</div>
+
+```typescript
+// app.module.ts
+import { UserMenuModule, InfoListItemModule } from '@pxblue/angular-components';
+...
+imports: [
+    UserMenuModule,
+    InfoListItemModule
+],
+...
+```
+
+```tsx
+let open = false;
+const items = [
+    {
+        title: 'Account',
+        icon: 'settings',
+    },
+    {
+        title: 'Contact Us',
+        icon: 'mail',
+    },
+    {
+        title: 'Log Out',
+        icon: 'logout',
+    }
+];
+```
+
+```html
+<pxb-user-menu avatarValue="HA" menuTitle='Sample Title' [(open)]="open">
+    <mat-nav-list pxb-body>
+        <pxb-info-list-item *ngFor="let item of items" [dense]="true" (click)="open=false">
+            <mat-icon pxb-icon>{{item.icon}}</mat-icon>
+            <div pxb-title>{{item.title}}</div>
+        </pxb-info-list-item>
+    </mat-nav-list>
+</pxb-user-menu> 
+```
+
+## API
+
+Parent element (`<pxb-user-menu>`) attributes:
+
+<div style="overflow: auto;">
+
+| @Input       | Description                                      | Type                    | Required | Default |
```

Comment:
If none of these inputs are required, what happens if none of them are supplied?

Reply:
The avatar and menu will render, but nothing will be shown in the avatar container. 

![image](https://user-images.githubusercontent.com/6538289/88283746-0a7a6c00-ccba-11ea-9320-59d8dbd7e403.png)

Reply:
![image](https://user-images.githubusercontent.com/6538289/88283805-2716a400-ccba-11ea-8240-847f85255cde.png)

Reply:
A little goofy, but I'll take a blue circle over nothing 👍